docs/integration/RequestResponse.mdx

```diff
@@ -26,8 +26,7 @@ curl -X POST -d "payload_based_on_input_schema" 'http://<ingress_domain><scenari
 
 ## OpenAPI
 
-After you deploy your `Request-Response` scenario an OpenAPI definition based on input and output schema you have provided in scenario properties will be available.
-You should see swagger based definition under `http://<scenario_name>/definition`
+After you deploy your `Request-Response` scenario an OpenAPI document (definition of the OpenAPI service) will be available at `http://<scenario_name>/definition`
```

Comment:
For each deployed `Request-Response` scenario, there will be available OpenAPI documentation. It will be reached at `http://<scenario_name>/definition`. 

WDYT?

Reply:
Corrected wording

Reply:
nope, it's not.